build.sbt

```diff
@@ -53,10 +53,10 @@ lazy val metricsScala = (project in file("metrics-scala"))
     name := "metrics4-scala",
     description := "metrics-scala for Scala " + CrossVersion.binaryScalaVersion(scalaVersion.value),
     libraryDependencies ++= Seq(
-      "io.dropwizard.metrics" % "metrics-core" % "4.0.1",
-      "io.dropwizard.metrics" % "metrics-healthchecks" % "4.0.1"
+      "io.dropwizard.metrics" % "metrics-core" % "4.0.3",
+      "io.dropwizard.metrics" % "metrics-healthchecks" % "4.0.3"
     ),
-    mimaPreviousArtifacts := Set("nl.grons" %% "metrics4-scala" % "4.0.1")
+    mimaPreviousArtifacts := Set("nl.grons" %% "metrics4-scala" % "4.0.3")
   )
```

Comment:
@erikvanoosten This change and all changes below should not have happened, right?

Reply:
That is correct, this should not have happened.
BTW, I did not review the other PRs yet. I'll try to find some time coming weekend.

Thanks! 👍

Reply:
Ok, I'll try to fix that in scala-steward.